t/APISIX.pm

```diff
@@ -217,6 +217,13 @@ if ($version =~ m/\/apisix-nginx-module/) {
 _EOC_
 }
 
+my $a6_ngx_vars = "";
```

Comment:
`a6_ngx_vars` looks strange.
Maybe `apisix_ngx_vars`  would be better?

Reply:
There is already a `a6_ngx_directives`, so `a6_ngx_vars` is fine